_maps/northstar.json

```diff
@@ -12,6 +12,9 @@
 	},
 	"space_ruin_levels": 0,
 	"space_empty_levels": 2,
+	"ignored_unit_tests": [
```

Comment:
added northstar to the ignore list bc it's not in rotation and i don't think anyone pays attention to it while PRing lol

this can ofc be removed if we ever actually start using northstar